packages/data/CHANGELOG.md

```diff
@@ -2,69 +2,76 @@
 
 ## Unreleased
 
+### New Features
+
+-   Added new `register` function for registering a standard `@wordpress/data` store definition ([#26655](https://github.com/WordPress/gutenberg/pull/26655)).
+-   Added new `createReduxStore` factory function that creates a data store definition for the provided Redux store options to use with `register` function ([#26655](https://github.com/WordPress/gutenberg/pull/26655)).
+-   Extended `select` and `dispatch` functions to accept a data store definition as their first param in addition to a string-based store name value [#26655](https://github.com/WordPress/gutenberg/pull/26655)).
+-   Extended `useDispatch` hook to accept a data store definition as their first param in addition to a string-based store name value [#26655](https://github.com/WordPress/gutenberg/pull/26655)).
```

Comment:
should we mention useSelect too?  or maybe just remove the line?

Reply:
`useDispatch` accepts directly store now so I included it, but `useSelect` doesn't as it exposes `select`. It's hard to tell what's the best way to document it 🤷‍♂️

Reply:
I see, I will fix it, thanks for catching it 👍